api/lib/opentelemetry/trace/tracer.rb

```diff
@@ -21,6 +21,15 @@ def current_span(context = nil)
         context.value(CURRENT_SPAN_KEY) || Span::INVALID
       end
 
+      # Returns a context containing the span, derived from the optional parent
+      # context, or the current context if one was not provided.
+      #
+      # @param [optional Context] context The context to use as the parent for
+      #  the returned context
+      def context_with_span(span, context = Context.current)
```

Comment:
Thoughts on making the second parameter named rather than positional? E.g. `parent_context: Context.current`